redis/src/test/scala/zio/redis/OutputSpec.scala

```diff
@@ -498,7 +498,182 @@ object OutputSpec extends BaseSpec {
             )
           )
           Task(XReadOutput.unsafeDecode(input)).either.map(assert(_)(isLeft(isSubtype[ProtocolError](anything))))
-        }
+        },
+        suite("XInfo stream commands output")(
```

Comment:
You can follow a similar naming practice that is established in existing cases and suites. You can name this just `xInfoStream`.